ADD-ONS-CHANGELOG.md

```diff
@@ -0,0 +1,59 @@
+# React Add-Ons Change Log
+
+As of React v15.5.0 we discontinued active maintenance of the React Addons
+packages. In previous versions of React we kept the version numbers of
+these packages in sync with the React version, and included updates to
+them in the main React `CHANGELOG.md`.
+
+Since the release versions of these packages are now out of sync with React,
+and we are no longer including them as a supported part of React, we have split
+out the latest changes into this separate change log.
+
+## Create Fragment
+
+### 15.5.3
+
+* Fix react-addons-create-fragment package to export correct thing. [See #9383](https://github.com/facebook/react/pull/9383).
+* Envify and collapse create-fragment UMD. [See #9383](https://github.com/facebook/react/pull/9383).
+
+## Create-React-Class
+
+### 15.5.2
+
+* Fix devDeps. [See #9383](https://github.com/facebook/react/pull/9383).
+* Add missing object-assign dependency. [See #9383](https://github.com/facebook/react/pull/9383).
+* Envify and collapse create-react-class UMD. [See #9383](https://github.com/facebook/react/pull/9383).
+
+## CSS Transition Group & Transition Group
+
+## Linked State Mixin
+
+### 15.5.2
+
+* Envify and collapse LinkedStateMixin UMD. [See #9383](https://github.com/facebook/react/pull/9383).
+
+## Linked Input
+
+### 15.5.1
+
+* Remove unused prop-types dep. [See #9383](https://github.com/facebook/react/pull/9383).
+* Fix change handling. [See #9383](https://github.com/facebook/react/pull/9383).
+* Fix and collapse LinkedInput UMD. [See #9383](https://github.com/facebook/react/pull/9383).
+
+## Pure Render Mixin
+
+### 15.5.2
```

Comment:
I tried to figure out either what version a change was released in, or at least what the latest version of each package is. The latest version of each of these is 15.5.2 and 15.5.1 respectively.